src/modules/tas_studio/editor/operation.rs

```diff
@@ -102,6 +103,26 @@ pub enum Operation {
         from: Option<String>,
         to: Option<String>,
     },
+    SetMaxAccelStart {
```

Comment:
What about something like
```suggestion
    SetMaxAccelOffsetStart {
```
and others similarly